sphinx.go

```
@@ -121,6 +121,12 @@ func (s *SphinxClient) Init(config *Config) error {
 
 	s.Config = *config
 
+	s.PoolInit()
+
+	return nil
```

This should return the error value from `PoolInit()`